src/QueryDescription/index.js

```diff
@@ -198,6 +219,18 @@ export function or(...conditions: Where[]): Or {
   return { type: 'or', conditions }
 }
 
+export function sortBy(sortColumn: ColumnName, sortOrder: SortOrder = 'asc'): SortBy {
+  return { type: 'sortBy', sortColumn, sortOrder }
+}
+
+export function take(count: number): Take {
+  return { type: 'take', count }
+}
+
+export function skip(count: number): Skip {
```

Comment:
I'd prefer if you call those three functions experimentalSortBy, experimentalTake, experimentalSkip -- the reason why is that I'm not entirely sure if mixing together query conditions and extra parameters like this is the correct API. I don't know what a better API would be, and I don't want to stop you from pushing such an awesome change until it can be perfect -- but I want to set the right expectation to 🍉 users that the exact API may change